class_exercise6.rb

```diff
@@ -3,16 +3,59 @@
 class PalindromicWords
 
   def initialize(word)
-    @word = word
+    @word = word.downcase
   end
 
   def palindromic_word
-    @word.eql?(@word.reverse) ? "This is a palindromic word" : "This is not a palindromic word"
+    @word.eql?(@word.reverse) ? "It is palindromic" : "It is not palindromic"
+  end
+  
+end
+
+#----------------Test------------------
+
+require 'minitest/autorun'
+
+class PalindromicWordsTest < Minitest::Test
+
+  def setup 
+    @word_true = PalindromicWords.new("oso")
```

Comment:
Please change this variable names, @word_true? it does'n make sense to me. 
@palindromic_word or something like that....
@any_word could be